### PR TITLE
feat: notes page with add and edit functionality

### PR DESCRIPTION
**Pull request by vanshitalavange**
Overview of this PR
- created notes page
- created side nav 
- created text editor ( toolbar is not functional)
- adding a note is functional 
- editing a note is functional
- change color of note card

Preview:
https://pencil-git-notes-page-vanshitalavange.vercel.app/

### CODE REVIEW

File: src/components/Sidebar/Sidebar.jsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    return <section className="sidebar">
+        <ul className="sidebar-list flex-column">
+            <li className="list-item flex-row active">
+                <span class="material-icons-outlined sidebar-list-icon app-icon">
+                    lightbulb
+                </span>
+                <p>Notes</p>
+            </li>
+            {/* <li className="list-item flex-row">
+                <span class="material-icons-outlined sidebar-list-icon">
```
Comment (ashishnnnnn) [inline src/components/Sidebar/Sidebar.jsx]: Commented code can be omitted..

File: src/pages/Auth/Signup.jsx [hunk 2]
```diff
@@ -1,3 +1,167 @@
...
+    password: "",
+    confirmPassword: "",
+  });
+
+  const isPasswordMatchedChecker = (event) => {
+    const confirmPassword = event.target.value;
+    if (newUserDetails.password !== "") {
+      if (confirmPassword === newUserDetails.password) {
+        setHelperMsg("Passwords matched 😃");
+        getFormInput(event, setNewUserDetails);
```
Comment (ashishnnnnn) [inline src/pages/Auth/Signup.jsx:32]: Liked the usages of emogi :)

File: src/pages/Home/Home.jsx [hunk 3]
```diff
@@ -1,6 +1,9 @@
 import "./Home.css";
-import { Link } from "react-router-dom";
+import { useNavigate } from "react-router-dom";
+import { useAuth } from "../../contexts";
 export const Home = () => {
+  const navigate = useNavigate()
+  const { userState: { loginStatus } } = useAuth()
   return (
     <section className="hero flex-row justify-center flex-wrap">
       <img className="hero-banner" src="assets/banner.png" />
```
Comment (ashishnnnnn) [inline src/pages/Home/Home.jsx]: Alt tag can be given here

**Review by ashishnnnnn — APPROVED** [review]

**Comment by ashishnnnnn** [comment]
Overall Code seem structured ..and more modular ..

**Review by RahulSingh9131 — APPROVED** [review]
* liked the idea of putting all routes in single file & then importing it
* css is clean and contains perfect use of declared variables
* perfect use of state variables from created context
only suggestion is to remove the commented code before merging.

**Pull request merged**